MAINTAINERS.md

```diff
@@ -2,22 +2,23 @@
 
 ## Maintainer
 
-- May override to merge a pull-request or push/revert commits.
-- Decides policies and architecture.
-- Designate reviewers and their dedicated subdirectories.
+- A committer that is also an arbiter for other committers in the Github/Git repositories.
+- May override merging criteria to merge/close a pull-request or to push/revert a commit.
 
-## Reviewer
+## Committer (previously Reviewer)
 
-- May reject a pull-request, which prohibits merging.
+- May reject a pull-request, which prohibits merging. "Veto"
 - May vote for an approval.
-- May merge a pull-request if it has enough number of approval-votes from reviewers or maintainers (2 or more).
-- The ability (vote/reject/merge) may be limited to a few subdirectories.
+- (Requires merge privliege) May merge a pull-request if it has enough number of approval-votes from reviewers or maintainers (2 or more).
```

Comment:
I think this is a typo. privliege -> privilege.

Reply:
Fixed. Thanks!